config/mappings/nstf_mapping_parts/nstf_r2_scored_criteria.py

```diff
@@ -149,7 +149,7 @@
                 "name": "Targeted criteria",
                 "themes": [
                     {
-                        "id": "targeted_criteria",
+                        "id": "targeted_criteria-current",
```

Comment:
We're ok with a mix of _ and - compound id names? I suspect this may cause issues with any kind of concatenation interpretations :/